### DISCUSSION TITLE
How to determine recording state?

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#1699] [Technical Support & Help]
Hi, I'm attempting to script things and have discovered there's no means to determine recording state (enabled/disabled).
I've built Jamulus 3.7.0 on Fedora 33 with "CONFIG+=noupcasename". I run `jamulus --server --nogui --discononquit --welcomemessage "Open Jam" --numchannels 10 --centralserver anygenre2.jamulus.io:22224 --serverinfo "Open Jam;AWS EAST;225" -R /data/jamulus --norecord -l /data/jamulus/jamulus-server.log` in a screen session.
I see STDOUT:
```
- server mode chosen
- no GUI mode chosen
- disconnect all clients on quit
- welcome message: Open Jam
- maximum number of channels: 10
- central server: anygenre2.jamulus.io:22224
- server info: Open Jam;AWS EAST;225
- recording directory name: /data/jamulus
- recording will not be enabled
- logging file name: /data/jamulus/jamulus-server.log
Recording state: disabled
 *** Jamulus, Version 3.7.0
 *** Internet Jam Session Software
 *** Released under the GNU General Public License (GPL)
Server Registration Status update: Registration requested
Server Registration Status update: Registered
Recording state: enabled
Recording state: disabled
```

Changing recording state with `kill -s SIGUSR2 $(pidof jamulus)` successfully changes state but that change is not logged in the log file, just to STDOUT.

Is this a bug, or a feature request?

**Comment by hoffie** [comment]
Hi,

>  that change is not logged in the log file, just to STDOUT.

Hrm, I'm wondering if we should align what goes to STDOUT with what goes to the log file. Not sure about the history/initial intentions though.

In any case, you should be able to redirect stdout to a (separate) log file as a workaround.

> Is this a bug, or a feature request?

Maybe a mixture. :)

**Reply by rdica** [reply]
Interesting, the command output is STDERR, I had to redirect
`2> logfile`
since neither 
`> logfile` nor `1> logfile`
failed to write to logfile.

Thanks!
